src/architecture/msgPayloadDefCpp/CameraModelMsgPayload.h

```diff
@@ -0,0 +1,43 @@
+/*
+ ISC License
+
+ Copyright (c) 2016-2018, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
:small_red_triangle_down: This copyright statement is incorrect.

Reply:
done